scripts/motor_stats.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+# To run this script set PYTHONPATH environment variable to register custom motor_stats message. 
+# Run this in main directory:- export PYTHONPATH=$PYTHONPATH:`pwd`/build/ardupilot_gazebo-msgs_genmsg/python/
+
+# Basic setup for dronecan:-
+# sudo modprobe vcan
+# sudo ip link add dev vcan0 type vcan
+# sudo ip link set up vcan0
+# To check:- ip link show vcan0
```

Comment:
We should be able to run this without needing to use `sudo`?

For example:
- https://github.com/srmainwaring/ardupilot_gazebo-1/blob/wips/wip-dronecan/scripts/dronecan_sensor.py

(that script following the examples in the dronecan repo and tutorials)